src/Microsoft.Health.Dicom.Core/Features/Store/StoreService.cs

```diff
@@ -115,6 +115,7 @@ public class StoreService : IStoreService
                         long len = length.GetValueOrDefault();
                         // Update Telemetry
                         _storeMeter.InstanceLength.Record(len);
+                        _dicomRequestContextAccessor.RequestContext.TotalDicomEgressToStorageBytes += len;
```

Comment:
Is this Ingress or egress? isn't egress is any data being downloaded or transferred out of the storage account.